openlibrary/macros/Pager.html

```diff
@@ -1,4 +1,4 @@
-$def with (page, num_found, results_per_page=20)
+$def with (page, num_found, results_per_page=1)
```

Comment:
Why was this change made?

Reply:
That's my bad, I was testing the pager layout and was forcing multiple pages, forgot to revert that. I will reset it as soon as I'm available.

Reply:
@sbwhitt I went ahead and removed that change so @jimchamp can finish his review sooner :)

Reply:
Thank you @RayBB !

Reply:
Thanks, all.